### PR TITLE
Fetch intro

### PR DESCRIPTION
**Pull request by shayla11**
Finished the Fetch intro tutorial of adding a button that calls to the server to get a message!

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ function addRandomMovieQuote() {
     const movieQuote = movieQuotes[Math.floor(Math.random() * movieQuotes.length)];
 
     // Add it to the page.
-    const movieQuoteContainer = document.getElementById('movie-quote-container');
+    const movieQuoteContainer = document.getElementById('quote-container');
     movieQuoteContainer.innerText = movieQuote;
 }
+
+async function getRandomMessage() {
+    const response = await fetch('/data');
```
Comment (oydang) [inline portfolio/src/main/webapp/script.js:43]: Does this actually compile? `fetch` isn't defined anywhere.

Comment (shayla11) [reply portfolio/src/main/webapp/script.js:43]: It compiles, but I'm not to well versed on its actually functionality beyond "fetching data". The tutorial didn't mention defining it, simply use "fetch()". Should it be defined somewhere in the code?

Comment (oydang) [reply portfolio/src/main/webapp/script.js:43]: Interesting! I guess you'll be defining it/configuring it later on in the walkthrough.

Comment (astrelni) [reply portfolio/src/main/webapp/script.js:43]: I believe `fetch` is one of those things provided by the browser.

**Pull request merged**